packages/static-build/test/fixtures/vue-v2/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "13-vue",
```

Comment:
Do you want to change the name here too? We could update this all later or just leave it.